docs/creating-editing-and-removing-specialist-document-types-and-finders.md

```diff
@@ -0,0 +1,161 @@
+
+3. In `search-api`, amend the value in the relevant schema in the [elasticsearch_types](https://github.com/alphagov/search-api/tree/main/config/schema/elasticsearch_types) directory. See [this](https://github.com/alphagov/search-api/pull/2320/commits/0f29e310581e30707eea7fe8c91063974636dbe2) commit for an example.
+
+4. Republish the finder, see step 3 in the [Deploy and publish](#deploy-and-publish) section above. It should suffice to run `search:update_schema` in `search-api`.
+
+# __Editing__ a specialist finder
+
+The [schema](https://github.com/alphagov/specialist-publisher/tree/main/lib/documents/schemas) files that define a specialist document, are also used to configure that document's specialist finder. See [this](https://github.com/alphagov/specialist-publisher/pull/1899/commits/925abc689119138a0e04e17d3610f8ae276773dd) commit for an example.
+
+# __Unpublishing__ a specialist finder
+
+**Important**: In order to remove a finder, you should ensure there is a clear business requirement in doing so, and that all associated documents have been correctly unpublished and migrated.
+If the code is not removed from all relevant repositories, there is a chance the finder will be inadvertently republished, for example by republishing all finders. You must therefore make sure the code is removed in order to keep the finder unpublished.
+
+The following steps are required to remove a finder:
+1. Unpublish the finder in all environments using the provided rake task:
+   ```
+   rake unpublish:redirect_finder["uk_market_conformity_assessment_bodies","https://redirection_link.gov.uk"]
+   ```
+2. Remove usages from `publishing-api`:
+   - Remove the format from [this list](https://github.com/alphagov/publishing-api/blob/main/content_schemas/formats/specialist_document.jsonnet#L2:L33) and [this list](https://github.com/alphagov/publishing-api/blob/main/content_schemas/allowed_document_types.yml).
+   - Remove the field definitions from [this file](https://github.com/alphagov/publishing-api/blob/main/content_schemas/formats/shared/definitions/_specialist_document.jsonnet).
+   - Remove the example from [this directory](https://github.com/alphagov/publishing-api/tree/main/content_schemas/examples/finder/frontend).
+   - Run `bundle exec rake build_schemas` to regenerate schemas.
+   
+   See [example commit](https://github.com/alphagov/publishing-api/pull/2706).
+
+3. Remove usages from `specialist publisher`. See [example commit](https://github.com/alphagov/specialist-publisher/pull/2588/files).
+4. Remove usages from `search-api`. See [example commit](https://github.com/alphagov/search-api/pull/2881/files).
+5. Remove any usages from `finder-frontend`.
```

Comment:
Maybe we can rephrase like: `Remove any usages from `finder-frontend` if applicable.` so it is less suggestive.